### DISCUSSION TITLE
Problem with Nested double quote and single quote command

### DISCUSSION THREAD

**Discussion by sorrentinov** [google/zx#730] [Help needed]
## Expected Behavior
I have two variables:

const sharePointSite = "https://mysharepoint.com/sites/MySite"
const sharePointListName = "MyList" 

I would in zx execute this command (using m65 cli) that return the existing list in the site filtered by Title:
$`m365 spo list list --webUrl ${sharePointSite} --properties Id,Title --filter "Title eq $sharePointListName" --output text`

I've tried on bash and it works fine:
m365 spo list list --webUrl https://mysharepoint.com/sites/MySite --properties Id,Title --filter "Title eq 'MyList'" --output text

## Actual Behavior

I receive this error:   _The expression "Title eq $'MyList'" is not valid._

## Steps to Reproduce the Problem

1. install m365  
1. use the command above

## Specifications

- Version: 7.2.3
- Platform: Mac

**Comment by antonmedv** [comment]
I didn't get it.
